### PR TITLE
add Location fields validation

### PR DESCRIPTION
**Pull request by sfc-gh-fhoffmann**

### CODE REVIEW

File: src/main/java/pl/gov/coi/pomocua/ads/Location.java [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+import javax.validation.constraints.NotNull;
 
 @Embeddable
 @NoArgsConstructor
 @AllArgsConstructor
 @Data
 public class Location {
+    @NotNull
     public String region;
+    @NotNull
```
Comment (sfc-gh-krzymkowski) [inline src/main/java/pl/gov/coi/pomocua/ads/Location.java:18]: How about `@NotBlank`?

**Review by sfc-gh-krzymkowski — APPROVED** [review]

File: src/main/java/pl/gov/coi/pomocua/ads/Location.java (hunk 1 above)
Comment (sfc-gh-fhoffmann) [reply src/main/java/pl/gov/coi/pomocua/ads/Location.java:18]: yes - sounds like a good idea

**Review by sfc-gh-tkaminski — APPROVED** [review]

File: src/main/java/pl/gov/coi/pomocua/ads/transport/TransportOfferSearchCriteria.java [hunk 2]
```diff
@@ -4,10 +4,12 @@
...
 import pl.gov.coi.pomocua.ads.Location;
 
+import javax.validation.Valid;
 import java.time.LocalDate;
 
 @Data
 public class TransportOfferSearchCriteria {
+    @Valid
     private Location origin;
     private Location destination;
```
Comment (sfc-gh-pnescior) [inline src/main/java/pl/gov/coi/pomocua/ads/transport/TransportOfferSearchCriteria.java:14]: Shouldn't it be `@Valid` as well?

Comment (sfc-gh-fhoffmann) [reply src/main/java/pl/gov/coi/pomocua/ads/transport/TransportOfferSearchCriteria.java:14]: true that, fixed

**Review by sfc-gh-pnescior — APPROVED** [review]

**Pull request merged**